projects/showcase/src/app/demo/examples/cluster-html.component.ts

```diff
@@ -57,7 +61,18 @@ const COLORS = ['#fed976', '#feb24c', '#fd8d3c', '#fc4e2a', '#e31a1c'];
       ></mgl-layer>
     </mgl-map>
   `,
-  styleUrls: ['./examples.css'],
+    styleUrls: ['./examples.css'],
+    standalone: true,
+    imports: [
+        MapComponent,
+        MglMapResizeDirective,
+        MapTestingHelperDirective,
+        GeoJSONSourceComponent,
+        MarkersForClustersComponent,
+        ClusterPointDirective,
+        forwardRef(() => ClusterPointComponent),
```

Comment:
Why is this using `forwardRef`? Is there a problem with circular dependency?

Reply:
This is part of the `ng generate @angular/core:standalone` schematics, will try to remove it too.

Reply:
I guess that it's because it's in this file after this component instead of before or in another file. I don't think the forwardRef is actually needed, but I might be wrong...

Reply:
Compiler is complaining about the declaration, my guess is the schematics fixed it with `forwardRef``

![CleanShot 2023-11-22 at 16 09 02](https://github.com/maplibre/ngx-maplibre-gl/assets/8985933/cc58e6cc-9065-4dcb-9ebc-057df61d041b)

Reply:
I removed `forwardRef` and changed the order of the components, the demo pages are working fine.